packages/amplify_datastore_plugin_interface/lib/amplify_datastore_plugin_interface.dart

```diff
@@ -80,15 +84,18 @@ abstract class DataStorePluginInterface extends AmplifyPluginInterface {
   }
 
   /// Configure AmplifyDataStore plugin with mandatory [modelProvider]
-  /// and optional datastore configuration properties including
+  /// and optional DataStore configuration properties including
+  ///
+  /// [errorHandler]: custom error handler code executed when DataStore encounters an unhandled error during its background operations
```

Comment:
Maybe this could be a bit clearer that we are expecting from the error handler. 

```suggestion
  /// [errorHandler]: Custom error handler function that may receive an [AmplifyException] object when DataStore encounters an unhandled error during its background operations
```

Reply:
Thanks updated with your suggestions!